models/staging/base/base_ga4__events.sql

```diff
@@ -20,8 +20,8 @@
 with source as (
     select
         {{ ga4.base_select_source() }}
-    from {{ source('ga4', 'events') }}
-    where cast(left(replace(_table_suffix, 'intraday_', ''), 8) as int64) >= {{var('start_date')}}
+    from {{ source('ga4', 'events') }}    
+    {{ select_date_range_base(start_date, end_date) }}
```

Comment:
Do we want the start_date to always be there? If so, this might interfere with the `lookback_window` and `static_incremental_days`. If okay, please document appropriately.